### ISSUE TITLE
Typo error "soved date format issue" and requesting code modification on my main.py with pybangla

### ISSUE THREAD

**Issue by khandakershahi** [saiful9379/pybangla#13]
First, thanks for building the pybangla package.

A typo error on https://github.com/saiful9379/pybangla/commit/05f9e895b916fdd60b167117be96a00eead9666a  "soved date format issue"

And a request, not a python coder. Can you please modify the code below so it recognizes the Bengali years?
```


import torch
from TTS.api import TTS
import gradio as gr

device = "cuda" if torch.cuda.is_available() else "cpu"

def generate_audio(text="তুমি কেমন আছো?"):
    tts = TTS(model_name='tts_models/bn/custom/vits-male').to(device)
    tts.tts_to_file(text=text, file_path="outputs/output.wav")
    return "outputs/output.wav"

demo = gr.Interface(
    fn=generate_audio,
    inputs=[gr.Text(label="Text"),],
    outputs=[gr.Audio(label="Audio"),],
    )


demo.launch()

```

 coqui-ai    /    TTS did provide me some code with mine but it don't prounice bengali year as "১৯৮৭-র" to "উনিশশো সাতাশি এর" rather just plain number in bengali.
 
 Here is th [link](https://github.com/coqui-ai/TTS/issues/3815)

Thanks.

**Comment by saiful9379** [comment]
@khandakershahi, if your problem is year check the **date_format** or you can use this normalizer, it will solve the Date Normalization as well.

**Comment by khandakershahi** [comment]
I don't know python coding. These codes I got from YouTube. If I know how to use with GUI by coding, I wouldn't ask you for help. 
In TTS GitHub, they help me with update code. I was hoping a similar thing. But not everybody has time to help or will. Still I ask. 

Thanks for your package. But I guess I won't be able to use it, as I don't know python coding. Neither someone created a tutorial on anywhere how to use it with GUI for the Bengali or May be in my search I didn't find it.

**Comment by saiful9379** [comment]
Please check the codebase it will solved your problem.
`pip install pybangla`

`import torch
from TTS.api import TTS
import gradio as gr

import pybangla
nrml = pybangla.Normalizer()

device = "cuda" if torch.cuda.is_available() else "cpu"
text = "তুমি কেমন আছো?"

text = nrml.text_normalizer(text)

def generate_audio(text=text):
    tts = TTS(model_name='tts_models/bn/custom/vits-male').to(device)
    tts.tts_to_file(text=text, file_path="outputs/output.wav")
    return "outputs/output.wav"

demo = gr.Interface(
    fn=generate_audio,
    inputs=[gr.Text(label="Text"),],
    outputs=[gr.Audio(label="Audio"),],
    )


demo.launch()`

**Comment by khandakershahi** [comment]
Thanks it works. Really appreciate it.
